plugin/web/extensions/geolocation/core/index.html

```diff
@@ -0,0 +1,21 @@
+<title></title>
+<style>
+  html,
+  body {
+    background: transparent;
+  }
+  .root {
+    background: #fff;
```

Comment:
Don't set the background here. If you do, when the widget loads it might flash white before the color we actually want.

Reply:
done

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6